spec/presenters/two_factor_auth_code/phone_delivery_presenter_spec.rb

```diff
@@ -53,6 +60,24 @@
   end
 
   describe '#fallback_links' do
+    it 'handles multiple locales' do
+      I18n.available_locales.each do |locale|
+        I18n.locale = locale
+        presenter_for_locale[locale].fallback_links.each do |html|
```

Comment:
Why not just call `presenter_with_locale` here and do away with the hash?

Reply:
thought of that but I wasn't a fan of assembling the hash for every (client) call.  You never know when 1 call turns into 10,000.

Reply:
Well, you wouldn't need to assemble the hash, you could just call the helper and use the result in your test. Also, this won't be used outside the tests so it looks to me like a good tradeoff for readability.

Since the hash is built with `let`, it would also get rebuilt for every example that uses it anyway.